public class JavaFail61 {
    public static void main(String[] args) {
//        c();

        for (int i = 0; i <= 10; i++) {

        }


    }
    public static void c() throws Exception{
        int i;
        System.out.println();
        throw new Exception();


    }
    public static void d() {
        int i;
    }
}
